test/linter/test-style.js

```diff
@@ -58,50 +60,99 @@ function orderFeatures(key, value) {
   return value;
 }
 
+/**
+ * Return a new feature object whose first-level properties have been
+ * ordered according to compareVersions, and so will be
+ * stringified in that order as well. This relies on guaranteed "own"
+ * property ordering, which is insertion order for non-integer keys
```

Comment:
This doesn't seem right, we do have "integer" keys like "10" as well as keys like "10.1". These are iterated in the wrong order, and they will be after putting them in a new object as well, right?

Does this lint fail when such versions are out of order?

Reply:
Description updated -- apparently there was also a typo in the script file so this wasn't updating properly.  What's odd though is that the ordering doesn't seem to be right, hmm...